pkgs/applications/editors/cosmic-edit/default.nix

```
@@ -24,21 +24,22 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cosmic-edit";
-  version = "unstable-2023-11-02";
+  version = "unstable-2023-11-10";
 
   src = fetchFromGitHub {
     owner = "pop-os";
     repo = "cosmic-edit";
-    rev = "ee2dea71e6e21967bc705046f9650407d07cdada";
-    sha256 = "sha256-27j9Imlyzwy2yjpNsWWcX0qW38ZxMDkht1Eaggr4NYY=";
+    rev = "c66a28320a5274d62d68a6d90567bcf4d3e9666c";
+    sha256 = "sha256-QmIF2QAKx/dImKLy2qXJHIBs+/JhbHXE4FTqR2FClrA=";
```

```suggestion
    hash = "sha256-QmIF2QAKx/dImKLy2qXJHIBs+/JhbHXE4FTqR2FClrA=";
```
The hash string already describes the algorithm

---

Should be good now.